Js/main.js

```diff
@@ -1,5 +1,12 @@
-const container = document.querySelector(".items-container");
+const data =require('./data.js')
 
+//filter By Caregorey
+const filterByCaregorey= ((SearchCategory)=>{
+  const SearchedArray= data.filter((ele)=>ele.category==SearchCategory)
+  //  return  displayData(SearchedArray)
```

Comment:
Yasmin said don't push commented code

Reply:
and another think I didn't notice it when i first review the code
take care of naming the argument and const
should be category not search xD